card/src/main/java/com/adyen/checkout/card/CardComponentParamsMapper.kt

```diff
@@ -101,6 +104,12 @@ internal class CardComponentParamsMapper(
         )
     }
 
+    private fun CardComponentParams.removeRestrictedCards(): CardComponentParams {
```

Comment:
This should be done in `CardConfiguration.getSupportedCardTypes` so that we have all the logic for supported card types in one function